<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IExportDatabase</class>
 <widget class="QDialog" name="IExportDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export database to GPX...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolPath">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/PathBlue.png</normaloff>:/icons/32x32/PathBlue.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Export Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkGpx11">
       <property name="text">
        <string>GPX 1.1 without extensions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkDel">
       <property name="text">
        <string>Delete content in export folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushAbort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>IExportDatabase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
